README.md

```diff
@@ -413,73 +418,125 @@ Response:
   + `409` - Invalid path / Resource already exists
   + `415` - Invalid file type
   + `413` - Request data too large
+  + `501` - Invalid action / Action-Resource type conflict
+
+
+- [2.2](#2.2) <a name='2.2'></a> **bulk**
+  > Create multiple resources with optional initial data, in a specified directory.
+
+  Parameters
+  + `resources` `json` -
+    Map of names to content of resources to be uploaded.
```

Comment:
Change to "Hash of names and content of resources to be uploaded." to improve clarity